CRM/Event/Form/ManageEvent/TabHeader.php

```diff
@@ -85,7 +85,9 @@ public static function process(&$form) {
       $tabs['reminder'] = ['title' => ts('Schedule Reminders'), 'class' => 'livePage'] + $default;
     }
 
-    $tabs['friend'] = ['title' => ts('Tell a Friend')] + $default;
+    if (function_exists('tellafriend_config')) {
+      $tabs['friend'] = ['title' => ts('Tell a Friend')] + $default;
+    }
```

Comment:
I'd think this could be done with `hook_civicrm_tabSet()`

Reply:
That specific one is a deprecated function scheduled for deletion in 5.78, but I fixed the other place for Events.